google/ggrc-core
# GGRC-6420 Add "Other Contacts" role for Control

# Issue description

Customer needs to have new built-in attribute "Other Contacts" for control object.

# Steps to test the changes

Check correctness of creation of new Control, updating existing Controls through the UI and Import/Export. Check Assessment generation with "Other Contacts" in assessment template.
Check "Other Contacts" people permissions.

# Solution description

Added new row into ACR, updated propagation tree.

# Sanity checklist


# Migration checklist

# PR Review checklist
